vivisect/parsers/elf.py

```diff
@@ -76,6 +80,26 @@ def makeRelocTable(vw, va, maxva, addbase, baseaddr):
         vw.setComment(va, tname)
         va += len(s)
 
+def makeFunctionTable(elf, vw, tbladdr, size, tblname, funcs, ptrs, baseaddr=0):
+    psize = vw.getPointerSize()
+    pfmt = e_bits.le_fmt_chars[psize]   #FIXME: make Endian-aware (needs plumbing through ELF)
```

Comment:
Oh boy, this fixme seems like something we should tackle right now.

Reply:
i'm completely ok with that....  done.